package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@soos-io/soos-sca",
+  "version": "0.1.0",
+  "description": "SOOS Security Analysis CI",
+  "main": "bin/index.js",
+  "scripts": {
+    "setup:install": "npm install",
+    "setup:clean-install": "npm ci",
+    "setup:update": "npx npm-check -u",
+    "setup:clean": "npx rimraf node_modules && npx rimraf package-lock.json",
+    "build": "tsc",
+    "build:clean": "npx rimraf build",
+    "format": "prettier ./src --check",
+    "format:fix": "prettier ./src --write",
+    "typecheck": "tsc --noEmit",
+    "check": "npm run format && npm run typecheck && npm outdated"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/soos-io/soos-sca.git"
+  },
+  "author": "SOOS",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/soos-io/soos-sca/issues"
+  },
+  "homepage": "https://github.com/soos-io/soos-sca#readme",
+  "dependencies": {
+    "@soos-io/api-client": "0.2.0-pre.5",
```

Comment:
once https://github.com/soos-io/soos-api-client/pull/9 this is merged, change and run npm i